src/main/java/com/github/vkorobkov/jfixtures/loader/Fixture.java

```diff
@@ -15,4 +19,11 @@ public Fixture(String name, Collection<FixtureRow> rows) {
         this.name = name;
         this.rows = Collections.unmodifiableCollection(rows);
     }
+
+    public Fixture mergeRows(Collection<FixtureRow> rows) {
```

Comment:
Can we rename `rows` with something like `newRows`? `mergingRows`?

Reply:
Why? Because it overlaps with `this.rows`?

Reply:
yes, looks a bit unclear `this.rows.stream(), rows.stream()`

Reply:
Fixed